macros/models/_sector/referral/platforms/paragraph_referral_rewards.sql

```diff
@@ -35,4 +37,39 @@ where
 {% if is_incremental() %}
     and {{incremental_predicate('evt_block_time')}}
 {% endif %}
+
+UNION ALL
+
+select
+    '{{blockchain}}' as blockchain
+    ,'paragraph' as project
+    ,'v1' as version
+    ,call_block_number as block_number
+    ,call_block_time as block_time
+    ,cast(date_trunc('day',call_block_time) as date) as block_date
+    ,cast(date_trunc('month',call_block_time) as date) as block_month
+    ,call_tx_hash as tx_hash
+    ,'NFT' as category
+    ,0x0000000000000000000000000000000000000000 as referrer_address
+    ,c.to as referee_address
+    ,{{native_currency_contract}} as currency_contract
+    ,0 as reward_amount_raw
+    ,c.contract_address as project_contract_address
+    ,row_number() over (partition by call_tx_hash order by call_trace_address) + 9999999 as sub_tx_id
+    ,tx."from" as tx_from
+    ,tx.to as tx_to
```

Comment:
I noticed something really odd: `call_tx_from` doesn't exist in any of the polygon `Contract_call_functionName` tables. At first I thought it might have been a bug with this specific table, but I checked a few others and none of them have those two columns. Is that a known limitation or something or a bug?

Reply:
Those 2 columns have only recently been added to the call and event tables.
So I believe it's not out there for all chains yet, it should be added in the near future.

Reply:
Ah I see. Is that documented anywhere? Works without it, just curious if there was a way I could that wasn't write the query -> open PR -> wait for CI to fail

Reply:
no docs yet I think, I believe the columns are only 'stealth launched' for a couple chains now.
Only way to know this is if you see it in the data explorer on Dune. It'll be properly announced when all chains are covered.